bosh-stemcell/spec/stemcells/ubuntu_bionic_spec.rb

```diff
@@ -16,6 +16,7 @@
       its(:content) { should match ' selinux=0' }
       its(:content) { should match ' cgroup_enable=memory swapaccount=1' }
       its(:content) { should match ' console=ttyS0,115200n8' }
+      its(:content) { should match ' console=hvc0' }
```

Comment:
This change will probably break other infrastructure providers. Consider scoping it to CloudStack.

Reply:
just ran a few tests and doesn't seem to break any tests although i would have expected it too break.

Reply:
@rkoster < you are right, I did ran the tests and the builds only for our infrastructure and forgot that; let me fix it.

Reply:
i tested it multiple times as i thought this would indeed brake other iaases. but no issues so far

Reply:
LGTM